Your_second_ASDF_converter.ipynb

```diff
@@ -0,0 +1,1142 @@
+    "the structure. There will be two converters to illustrate how to\n",
+    "handle multiple converters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7f25df85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir myconverters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bbc846f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir myconverters/src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1ca1d34",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir myconverters/src/myconverters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "978d413d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd myconverters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e94f6849",
+   "metadata": {},
+   "source": [
+    "Add the usual files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61da8bca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/myconverters/photo_id.py\n",
+    "class PhotoID:\n",
+    "    \"Holds Phot ID information\"\n",
+    "\n",
+    "    def __init__(self, last_name, first_name, image):\n",
+    "        \"expects a monochromatic numpy array for image\"\n",
+    "        self.last_name = last_name\n",
+    "        self.first_name = first_name\n",
+    "        self.photo = image\n",
+    "    \n",
+    "    def name(self):\n",
+    "        return self.last_name + ', ' + self.first_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e318f5c2",
+   "metadata": {},
+   "source": [
+    "Add a second class that references the first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ff6bd411",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/myconverters/traffic_citation.py\n",
+    "from .photo_id import PhotoID\n",
+    "\n",
+    "\n",
+    "class TrafficCitation:\n",
+    "    \"Record of a traffic violation\"\n",
+    "\n",
+    "    def __init__(self, ociffer, violation, date, time, photo_id):\n",
+    "        self.ociffer = ociffer\n",
+    "        self.violation = violation\n",
+    "        self.date = date\n",
+    "        self.time = time\n",
+    "        if not isinstance(photo_id, PhotoID):\n",
+    "            raise ValueError(\"not a PhotoID instance\")\n",
+    "        self.photo_id = photo_id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "482ba07e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/myconverters/converters.py\n",
+    "from asdf.extension import Converter\n",
+    "from .photo_id import PhotoID\n",
+    "from .traffic_citation import TrafficCitation\n",
+    "\n",
+    "\n",
+    "class PhotoIDConverter(Converter):\n",
+    "    tags = [\"asdf://stsci.edu/example-project/photo_id-*\"]\n",
```

Comment:
In the cookiecutter instructions, we specify project `myconverters`, but here we're using `example-project`

Reply:
The dangers of writing stuff that isn't tested. Fixed.